crates/nu-command/tests/commands/path/type_.rs

```diff
@@ -50,5 +62,15 @@ fn returns_type_of_existing_directory() {
         ));
 
         assert_eq!(actual.out, "file");
+
+        let actual = nu!(
+            cwd: dirs.test(), pipeline(
+            r#"
+                echo "~"
+                | path type
+            "#
+        ));
+
+        assert_eq!(actual.out, "dir");
```

Comment:
modulo some formatting, you should be able to simplify this test to this :wink: 
```suggestion

        let actual = nu!(r#"
            echo "~" | path type
        "#);

        assert_eq!(actual.out, "dir");
```